libs/net/models/Areas/Subscriber/Models/Content/ContentModel.cs

```diff
@@ -1,5 +1,6 @@
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNetCore.Http;
+using TNO.API.Areas.Subscriber.Content;
```

Comment:
What's this reference for?

Reply:
This is for some of the models like Product or License that have a copy in the `/Content` folder